bin/yb-ctl

```diff
@@ -1064,22 +1180,23 @@ class ClusterControl:
         return self.customize_flags(command_list, self.options.master_flags)
 
     def get_tserver_only_flags(self, daemon_id):
-        daemon_ip_address_str = str(daemon_id.get_ip_address())
+        def get_host_port(port_type):
```

Comment:
this local one could be `get_client_host_port` and fetch the relevant combo, so when we call it we can just use `PROTOCOL_TYPE_YSQL` as an arg